tests/test_files/path_config.yaml

```diff
@@ -0,0 +1,5 @@
+  # absolute paths
+  pdb_file: './test_files/4v6x.pdb'
+  mrc_keyword: '_randomrot'
+  output_dir: './test_files'
+  local_sim_dir: './TEM-simulator'
```

Comment:
@ninamiolane this seems fine to me. While the ioSPI code is agnostic to simulator, it needs to be tested on something concretely.